Graviton_monitoring_differences.md

```diff
@@ -0,0 +1,56 @@
+# AWS Graviton monitoring differences
+Processor architecture choices are made for various reasons such as compatibility, performance and newer application domain support. These choices imply that the tools supported on the processors would differ too. Listed below are some  differences in the tools supported on AWS Gravtion when compared to x86 processor architectures.
+
+## Utilities for obtaining processor stats
+|Processor	|x86	|Graviton2,3	|
+|---	|---	|---	|
+|CPU frequency listing	|*lscpu, /proc/cpuinfo, dmidecode*	|*dmidecode*	|
+|*turbostat* support	|Yes	|No	|
+|*i7z* Works	|Yes	|No	|
+|Performance monitoring tools	|_[VTune Profiler](https://www.intel.com/content/www/us/en/developer/tools/oneapi/vtune-profiler.html) and [PCM](https://github.com/opcm/pcm), [Linux perf](https://www.brendangregg.com/perf.html)_	|_[Linux perf](https://www.brendangregg.com/perf.html), [Arm Forge](https://developer.arm.com/Tools%20and%20Software/Arm%20Forge)_	|
+
+More details on how to use Linux perf utility on AWS Graviton processors is available [here](https://github.com/aws/aws-graviton-getting-started/blob/main/optimizing.md#profiling-the-code).
+
+**Note:** On AWS Graviton 2/3 processors, Turbo isn’t supported. So, utilities such as ‘turbostat’ aren’t supported/ relevant for Arm architecture (and not on AWS Graviton processor either). Also, tools such as *[i7z](https://code.google.com/archive/p/i7z/)* for discovering CPU frequency, turbo, sockets and other information are only supported on Intel architecture/ processors.
+
+## Sample output
+#### Using lscpu utility
```

Comment:
I feel we are missing the flow... not sure why is lscpu mentioned here. we can say something like, lscpu can be used to get architecture details, cache levels and sizes, and HW capabilities via flags. However, it doesn't provide the cpu clock for arm/aarch64 systems for which we can use "dmidecode". then talk about "dmidecode'': how to run and what to look for etc...

Reply:
We can explicitly mention that in words or just put it in a table. So, I put the list of tools in a table to summarize quickly. The idea is that CPU frequency can be listed by utilities such as lscpu for x86 architectures but not for Graviton. So, for that we use dmidecode.